tests/php/ORM/DataObjectTest.php

```diff
@@ -2671,4 +2672,36 @@ public function testDBObjectEnum()
         $vals = ['25.25', '50.00', '75.00', '100.50'];
         $this->assertSame(array_combine($vals ?? [], $vals ?? []), $obj->dbObject('MyEnumWithDots')->enumValues());
     }
+
+    public function testGetDatabaseBackedField()
+    {
+        $dataObjectClass = new DataObject();
+        $method = new ReflectionMethod($dataObjectClass, 'getDatabaseBackedField');
+        $method->setAccessible(true);
+        $class = new Team([]);
+
+        $databaseBackedField = $method->invokeArgs($class, ['Captain.FavouriteTeam.Title']);
+        $this->assertSame('Captain.FavouriteTeam.Title', $databaseBackedField);
```

Comment:
Please also include a test for `'Captain.FavouriteTeam.Title.Plain'`

Reply:
Updated